### DISCUSSION TITLE
倒排索引搜索异常

### DISCUSSION THREAD

**Discussion by cst4049** [apache/doris#25104] [A - General / Q&A]
doris-2.0.0-alpha

MySQL select * from table where log MATCH_ALL 'redis ERROR' limit 10;
ERROR 1105 (HY000): errCode = 2, detailMessage = [INTERNAL_ERROR]failed to initialize storage reader. tablet=20621.1090931871.a24a0a81f3964cb4-e88c77d1c9a848b9, res=[E-6002]Inverted index read null bitmap error occurred,

部分分区可用， 搜索到最新的分区会报错

**Comment by LemonLiTree** [comment]
Upgrade to the latest 2.0.2, recreate the table and index and try again
